index.html

```diff
@@ -1,237 +1,258 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Raf's Portfolio</title>
+  <title id="Portfolio">Raf's Portfolio</title>
   <link rel="preconnect" href="https://fonts.googleapis.com">
   <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
   <link
     href="https://fonts.googleapis.com/css2?family=Inter&family=Crete+Round&family=Poppins&family=Roboto:wght@700&display=swap"
     rel="stylesheet">
   <link rel="stylesheet" href="styles.css">
 </head>
+
 <body>
   <header>
     <nav class="nav_bar flex">
       <h1 class="nav_brand">Rapheal Okonta</h1>
       <ul class="nav_ulist flex">
-        <li class="nav_link"><a href="#">Portfolio</a></li>
-        <li class="nav_link"><a href="#">About</a></li>
-        <li class="nav_link"><a href="#">Contact</a></li>
-        <li class="nav_link_email"><a href="#"><img src="image folder/Disabled.png" alt=""></a></li>
+        <li class="nav_link"><a href="#Portfolio">Portfolio</a></li>
+        <li class="nav_link"><a href="#About">About</a></li>
+        <li class="nav_link"><a href="#Contact">Contact</a></li>
```

Comment:
- [ ] At the moment only navigating to the `About` page is working. This is is because is the only section that has a matching `id`. Please consider adding the respective ids to the `Portfolio` and `Contact` sections. Remember that content inside a `<nav>` tag is expected to help navigate through your page. Making this change will improve this navigation, ending up with a more accessible website! 🤓 🛠️